contrib/google.golang.org/grpc/client.go

```diff
@@ -70,9 +70,20 @@ func StreamClientInterceptor(opts ...Option) grpc.StreamClientInterceptor {
 		fn(cfg)
 	}
 	return func(ctx context.Context, desc *grpc.StreamDesc, cc *grpc.ClientConn, method string, streamer grpc.Streamer, opts ...grpc.CallOption) (grpc.ClientStream, error) {
+		var methodKind string
+		if desc != nil {
```

Comment:
If we go by this logic, `cc` could also be `nil`. And if `desc` is ever `nil`, it will cause problems elsewhere too. The upstream package [doesn't seem to care](https://github.com/grpc/grpc-go/blob/0574097d679347aa1d3a3dc1590e0bc1decbc0c0/stream.go#L694) so why should we?

Reply:
We don't seem to be using `cc` in this function. 
Your argument makes sense, but In general, I try to follow the [robustness principle](https://en.wikipedia.org/wiki/Robustness_principle) and be liberal in what values we accept.
I think it's generally better if the library we're targeting makes the decision whether to panic or not. For all we know, they might consider that line a bug for not sanitizing possibly `nil` input. Maybe it'll be fixed in the future, and then we have an integration that causes panics on input that the library handles.
What do you think?

Reply:
Sure. Very good points. Please keep it then.